doc/source/WorkingWithScenes/LightLinking/screengrab.py

```diff
@@ -0,0 +1,130 @@
+# BuildTarget: images/interfaceDefaultLightPlug.png
+# BuildTarget: images/interfaceLinkedLightsAttribute.png
```

Comment:
It's there, it's just not immediately visible, because the file name is composed in L65-L81 of `screengrab.py`. It's a leftover from a hack that isolates each section of the Scene Inspector.

Reply:
Sorry I'd missed that...